client/src/components/App/Content/InvestigationForm/TabManagement/ClinicalDetails/SymptomsFields/SymptomsFields.tsx

```diff
@@ -69,6 +69,8 @@ const SymptomsFields: React.FC<Props> = (props: Props): JSX.Element => {
                                             onChange: (e, value) => {
                                                 handleSymptomsDateDataChange().then((result) => {
                                                     if(result.value) {
+                                                        !didSymptomsDateChangeOccur &&
+                                                            setDidSymptomsDateChangeOccur(true);
```

Comment:
both functions looks similar, can't you create one function for both?

Reply:
fixed